module/zfs/dbuf.c

```diff
@@ -3989,6 +4162,22 @@ EXPORT_SYMBOL(dmu_buf_set_user_ie);
 EXPORT_SYMBOL(dmu_buf_get_user);
 EXPORT_SYMBOL(dmu_buf_get_blkptr);
 
+#include <linux/mod_compat.h>
```

Comment:
We're now using this header in a couple files, let's move the include in to the __KERNEL section of `zfs_context.h`.

Reply:
Will this still need to happen if I move the dbuf cache flushing to the arc flush function?

Reply:
No, the header's just needed for `module_param_call()`.